config/static.go

```diff
@@ -24,65 +24,66 @@ type (
 		ExactVersion string
 	}
 
-	//UserCfgStaticCfg contains
-	UserCfgStaticCfg struct {
-		UpdateCheckFrequency *int `yaml:"UpdateCheckFrequency,omitempty"`
-	}
-
 	//MongoDBStaticCfg contains the means for connecting to MongoDB
 	MongoDBStaticCfg struct {
-		ConnectionString string        `yaml:"ConnectionString"`
-		AuthMechanism    string        `yaml:"AuthenticationMechanism"`
-		SocketTimeout    time.Duration `yaml:"SocketTimeout"`
+		ConnectionString string        `yaml:"ConnectionString" default:"mongodb://localhost:27017"`
```

Comment:
Every value going forward should have a new `default` tag. For clarity, even if the default value would be the same (e.g. empty string, 0, or empty array) we should define it explicitly so there's no questioning.  The `github.com/creasty/defaults` package takes care of interpreting these tags.